packages/plugin-analytic-google/src/GoogleAnalytic.js

```diff
@@ -54,19 +52,17 @@ export default class GoogleAnalytic extends AbstractAnalytic {
    *        analytic.
    */
   hit(data) {
-    this.deferHitAfterLoad(() => {
-      this._window
-        .getWindow()
-        .ga(
-          data.method || 'send',
-          data.type || 'event',
-          data.category || 'undefined',
-          data.action || 'undefined',
-          data.label,
-          data.value,
-          data.fields
-        );
-    });
+    this._window
```

Comment:
The method is not safe for server side because getWindow returns undefined. The last version before implementation of defer hit was safe https://github.com/seznam/IMA.js-plugins/blob/bf729e6e8d761c6fd914ee62599b1b227620a01f/packages/plugin-analytic-google/src/GoogleAnalytic.js#L74.

Reply:
All public methods must be safe for server side.

Reply:
Utilized `this.isEnabled()` method in `hit` and `hitPageView` methods before requesting window object.